lab/user-manual.md

```diff
@@ -0,0 +1,250 @@
+- Disk storage used, and total and available RAM 
+
+In the top right corner, you can find the help button that will provide you with
+helpful documentation and links to the Memgraph community.
+
+All the Memgraph Lab sections are listed in the left side menu, below which you
+can find the [Layout](#layout) options and Memgraph Lab version. 
+
+## Query Execution
+
+In this section, you can write and run queries, as well as see their tabular or
+graphical results. 
+
+### Cypher Editor
+
+Here is where you write and run your Cypher queries. A keyword suggestion tool
+can help you with clause completion and give information about signatures and
+parameters. If you need help writing Cypher queries, check out the [Cypher
+manual](/cypher-manual).
+
+<img src={require('./data/user-manual/intelisense.png').default} className={"imgBorder"} style={{marginBottom:'20px'}}/>
+
+Once you **Run** a query (by clicking a button or pressing **CTRL** +
+**Enter**), you can **Cancel** it, but if the query has already
+reach MemgraphDB, the action won't actually stop the query's execution. In the
+case of running complex algorithms on a large dataset, you need to be a bit
+patient and wait for Memgraph to complete running the query. 
+
+When the query has executed or failed, below the Cypher editor, you can see a
+*Query successful* message or an error. In the case of successful execution,
+you can also find out query execution time, rows the query generated as well as
+nodes and relationship the query returned (if any).
+
+Here you can also copy the query to your clipboard or add it to an existing or
+new query collection. 
+
+### Data results 
+
+Depending on the results generated by the query, they can be shown in a
+table, as a graph, or both. 
+
+Here, you can also **Download Results** in JSON format. 
+
+Table rows can be expanded to show additional information about entities.
+
+<img src={require('./data/user-manual/rows.png').default} className={"imgBorder"} style={{marginBottom:'20px'}}/>
+
+When results are shown as a graph, you can click on each node or relationship to
+see additional information. You can also **Expand** a node to see its
+relationships, **Collapse** a node to hide its relationships, or **Hide** the node from
+the canvas. 
+
+<img src={require('./data/user-manual/graph-results.png').default} className={"imgBorder"} style={{marginBottom:'20px'}}/>
+
+In the bottom left corner, you can **Enable physics**, that is, make nodes interact
+with each other, by pulling away or closing in to one another, depending on the
+strength of the relationships between them. 
+
+In the top right corner of the graph you can open **Graph Preferences** and set
+the collision radius that dictates the margin radius for each node from its
+center, the repel force that dictates how strongly nodes repel each other, and
+the link distance that dictates the minimum required distance between two
+connected nodes. 

```

Comment:
Image showing this actually also shows a known rendering bug (relationships not ending with arrows - they are seen behind nodes). It's not something that necessarily needs to be changed, just a suggestion.

Reply:
i don't think i care? we're not perfect :)